src/server.h

```diff
@@ -0,0 +1,32 @@
+/***********************************************************************
+ *
+ * Copyright (c) 2021
+ * IoTech Ltd
+ *
+ **********************************************************************/
+
+#ifndef BLE_SIM_SERVER_H
+#define BLE_SIM_SERVER_H
+
+#include <dbus/dbus.h>
+
+#include "defines.h"
+#include "service.h"
+
+typedef struct server_t
+{
+  char * object_path; //dbus object path
+  service_t *services;
+  
+} server_t;
+
+server_t *server_new (const char *object_path, service_t *services);
```

Comment:
Doxygen comments should be added for all functions

Reply:
This file has been removed in the current branch I am working on so little point in adding doxygen